packages/frontend/lib/congregation/congregation-information-board.ts

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+
+import { backendRoutes } from "../config";
+import { requestOptions } from "../request-options";
+import {
+  InformationBoardItem,
+  informationBoardItemSchema,
+  NewInformationBoardItem,
+} from "../types/dashboard";
+
+export async function addInformationBoardItem(item: NewInformationBoardItem) {
+  try {
+    await axios.post(
+      backendRoutes.congregation.informationBoard.add,
+      item,
+      requestOptions(),
+    );
+  } catch (err) {
+    throw err;
```

Comment:
Remove this whole try-catch. If the code throws you just re-throw the error which is redundant.

Reply:
ok